docs/reference/python/index.md

```diff
@@ -0,0 +1,21 @@
+# Getting Started
+
+We use Python 3 to write the majority of our software at UBC Sailbot. Therefore, it is critical that you get familiar
+with the language and be able to use it to get your assigned tasks completed.
```

Comment:
@hhenry01 do you imagine that Network Systems will use Python for the new project? If not, we should update this to say that it is used by Pathfinding and Controls

Reply:
Do we really need to be specific here? Subteams might change in the future, and I feel like we can just direct new members which resources to consult on here.

Reply:
I just wanted to get across that it might not be necessary for all software members to know python